fc-community-backend/apps/api/src/factchain-core/env.ts

```diff
@@ -12,6 +12,7 @@ const DEFAULT_CONFIG: Config = {
   AWS_SECRET_ACCESS_KEY: "",
   AWS_REGION: "",
   AWS_BUCKET: "",
+  BACKEND: "",
```

Comment:
is this a private key? if so any reason not to use OWNER_PKEY instead? Seems like it's valid to use it here.

Reply:
shhhhh 🤣